pkg/services/publicdashboards/api/middleware.go

```diff
@@ -28,14 +28,14 @@ func SetPublicDashboardOrgIdOnContext(publicDashboardService publicdashboards.Se
 	}
 }
 
-// Adds public dashboard flag on context
+// SetPublicDashboardFlag Adds public dashboard flag on context
 func SetPublicDashboardFlag(c *models.ReqContext) {
 	c.IsPublicDashboardView = true
 }
 
-// Middleware to enforce that a public dashboards exists before continuing to
-// handler
-func RequiresValidAccessToken(publicDashboardService publicdashboards.Service) func(c *models.ReqContext) {
+// RequiresExistingAccessToken Middleware to enforce that a public dashboards exists before continuing to handler. This
+// method will query the database to ensure that
+func RequiresExistingAccessToken(publicDashboardService publicdashboards.Service) func(c *models.ReqContext) {
```

Comment:
I think if we change the name of this method we need to update enterprise to call the new name.

Reply:
Thanks! I did the change but forgot to commit it, it is here https://github.com/grafana/grafana-enterprise/pull/4024